change/@ni-nimble-components-c0f72f9f-b879-43ad-bc5b-ee8b74e2700f.json

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
Have some questions @atmgrifter00 will bring up in the morning but I can join later to discuss if needed:
- In the Apache Arrow format does the metadata need to be included?
  
  It does not seem relevant to the render process based on discussions of the other table format. Also the apache arrow format cannot represent any JS value, i.e. `metadata: unknown[]` could be a class, function, dom element, etc.
  
  Should we use a format like `{table: Table<>, metadata?: unknown[]}` so any JS value is allowed?
  Should we only allow metadata to be possible arrow values? (breaking change)
  Should we force a type on metadata? (i.e. string, braking change)
- In the section on geoarrow and arquero my takeaway from the description was that the usage was "cumbersome" and we want to "avoid dependencies". What I want to understand is if those tools help make more maintainable code over time with good performance. It is not clear to me that those are behaviors of what was evaluated in the spec.
  - What I want to know if we have a clear evaluation of is if the high-level filter and query features of arquero can be used to manipulate data efficiently while also letting us use higher-level patterns for writing algorithms cleanly.
  - I created a stackblitz that sets up both libraries together that can be used to experiment with more easily: https://stackblitz.com/edit/geoarrow-worker-arquero-demo?file=src%2Fmain.ts&terminal=dev